Base/usr/share/man/man1/js.md

```diff
@@ -11,7 +11,7 @@ $ js [options...] [script.js]
 ## Description
 
 `js` evaluates JavaScript programs using the LibJS engine. If you pass it a path
-to a script file, it will execute that script. Otherwise, it enters the
+to a valid JavaScript file, it will execute that script. Otherwise, it enters the
```

Comment:
No. If you pass it a path to script, it executes that, otherwise it enters REPL — is what this sentence is saying. It won't enter REPL if you pass it an invalid JavaScript file for sure. Maybe changing it to "attempt to execute that script" would make sense.